plugins/module_utils/network/nxos/rm_templates/ospfv2.py

```diff
@@ -0,0 +1,1099 @@
+        command = "no {0}".format(command)
+    else:
+        external_lsa = max_metric.get("router_lsa", {}).get("external_lsa", {})
+        include_stub = max_metric.get("router_lsa", {}).get("include_stub", {})
+        on_startup = max_metric.get("router_lsa", {}).get("on_startup", {})
+        summary_lsa = max_metric.get("router_lsa", {}).get("summary_lsa", {})
+        if external_lsa:
+            command += " external-lsa"
+            if external_lsa.get("max_metric_value"):
+                command += " {max_metric_value}".format(**external_lsa)
+        if include_stub:
+            command += " include-stub"
+        if on_startup:
+            command += " on-startup"
+            if on_startup.get("wait_period"):
+                command += " {wait_period}".format(**on_startup)
+            if on_startup.get("wait_for_bgp_asn"):
+                command += " wait-for bgp {wait_for_bgp_asn}".format(
+                    **on_startup
+                )
+        if summary_lsa:
+            command += " summary-lsa"
+            if summary_lsa.get("max_metric_value"):
+                command += " {max_metric_value}".format(**summary_lsa)
+
+    return command
+
+
+def _tmplt_area_nssa(area):
+    nssa = area["nssa"]
+    command = "area {area_id} nssa".format(**area)
+    if nssa.get("set") is False:
+        command = "no {0}".format(command)
+    else:
+        for attrib in [
+            "no_summary",
+            "no_redistribution",
+            "default_information_originate",
+        ]:
+            if nssa.get(attrib):
+                command += " {0}".format(attrib.replace("_", "-"))
+    return command
+
+
+def _tmplt_area_nssa_translate(area):
+    translate = area["nssa"]["translate"]["type7"]
+    command = "area {area_id} nssa translate type7".format(**area)
+    for attrib in ["always", "never", "supress_fa"]:
+        if translate.get(attrib):
+            command += " {0}".format(attrib.replace("_", "-"))
+    return command
+
+
+def _tmplt_area_ranges(arange):
+    command = "area {area_id} range {prefix}".format(**arange)
+    if arange.get("not_advertise") is True:
+        command += " not-advertise"
+    if "cost" in arange:
+        command += " cost {cost}".format(**arange)
+    return command
+
+
+def _tmplt_area_ranges(arange):
+    command = "area {area_id} range {prefix}".format(**arange)
+    if arange.get("not_advertise") is True:
+        command += " not-advertise"
+    if "cost" in arange:
+        command += " cost {cost}".format(**arange)
+    return command
+
+
+def _tmplt_summary_address(proc):
+    command = "summary-address {prefix}".format(**proc)
+    if proc.get("tag"):
+        command += " tag {tag}".format(**proc)
+    elif proc.get("not_advertise"):
+        command += " not-advertise"
+    return command
+
+
+def _tmplt_area_stub(area):
+    stub = area["stub"]
+    command = "area {area_id} stub".format(**area)
+    if stub.get("set") is False:
```

Comment:
is it required to explicitly check for False
```suggestion
    if not stub.get("set"):
```